unit-tests/live/frames/test-fps.py

```diff
@@ -3,7 +3,7 @@
 
 # test:device each(D400*)
 # test:donotrun:!nightly
-# test:timeout 250
+# test:timeout 154
```

Comment:
test took 140.40848207473755 seconds
Do you think 154 is enough?
Since LibCI default timeout is 250 (IIRC) we can drop this line at all, we only need it when we want to increase , right?

Reply:
On win it took more
`test took 151.15837216377258 seconds`

Reply:
This is from my run with `nightly` context, the PR run skip this test :)

Reply:
Removed the line. Looks OK in CI